app/controllers/chores_controller.rb

```diff
@@ -0,0 +1,48 @@
+class ChoresController < ApplicationController
+  def create
+    begin
+      params.require(:chore).permit(:title, :hideout_id)
+    rescue ActionController::ParameterMissing
+      render status: 400
```

Comment:
This should be `params.require([:chore, :title, :hideout_id])`. The rescue should wrap the entire body of this function. You also need an `end` after the rescue and render. Also, add a `return` before the status since we want the function to end early in this rescue.

Reply:
check now